paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletViewModel.kt

```diff
@@ -98,6 +100,13 @@ internal class WalletViewModel @Inject constructor(
             is LinkLaunchMode.Authentication -> null
         }
 
+    private val LinkLaunchMode.paymentSelectionHint: String?
+        get() = (this as? LinkLaunchMode.PaymentMethodSelection)?.hint
+            ?.takeIf {
+                configuration.enableLinkPaymentSelectionHint ||
+                    FeatureFlags.forceEnableLinkPaymentSelectionHint.isEnabled
+            }
```

Comment:
[no action] is there any reason not to show this on `Full` flows, that also display the wallet? there's no use-case today but we can probably avoid that check